setup_jdk21_and_mvn.sh

```
@@ -1,51 +1,93 @@
+# Check if sdk command is available (in subshell)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)